sklearn/utils/tests/test_extmath.py

```diff
@@ -545,15 +545,8 @@ def naive_mean_variance_update(x, last_mean, last_variance,
     A1 = np.full((n_samples // 2, n_features), x2, dtype=np.float64)
     A = np.vstack((A0, A1))
 
-    # Older versions of numpy have different precision
-    # In some old version, np.var is not stable
-    if np.abs(np_var(A) - two_pass_var(A)).max() < 1e-6:
-        stable_var = np_var
-    else:
-        stable_var = two_pass_var
-
```

Comment:
Original PR indicates that this was for numpy < 1.9 so we should indeed be fine. https://github.com/scikit-learn/scikit-learn/pull/5104